sdk/communication/AzureCommunication/Tests/CommunicationUserCredentialTests.swift

```diff
@@ -101,11 +101,12 @@ class CommunicationUserCredentialTests: XCTestCase {
     func test_RefreshTokenProactively_TokenAlreadyExpired() throws {
         let expectation = XCTestExpectation()
 
-        let userCredential = try CommunicationUserCredential(
+        let option = CommunicationTokenRefreshOptions(
```

Comment:
Might want to rename this to something a bit more explicit (e.g. `tokenRefreshOptions`)